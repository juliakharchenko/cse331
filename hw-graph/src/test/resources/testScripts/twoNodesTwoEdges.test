# Tests if two nodes in a graph can have an edge pointing from node 1 to node 2 and
# an edge pointing from node 2 to node 1

# Create a graph
CreateGraph twoNodeGraph

# Add nodes
AddNode twoNodeGraph a
AddNode twoNodeGraph b

# Add edges
AddEdge twoNodeGraph a b AB
AddEdge twoNodeGraph b a BA

# Print the nodes in the graph and the outgoing edges from a and b
ListNodes twoNodeGraph
ListChildren twoNodeGraph a
ListChildren twoNodeGraph b