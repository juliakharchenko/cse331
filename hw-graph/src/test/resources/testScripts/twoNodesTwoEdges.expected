# Tests if two nodes in a graph can have an edge pointing from node 1 to node 2 and
# an edge pointing from node 2 to node 1

# Create a graph
created graph twoNodeGraph

# Add nodes
added node a to twoNodeGraph
added node b to twoNodeGraph

# Add edges
added edge AB from a to b in twoNodeGraph
added edge BA from b to a in twoNodeGraph

# Print the nodes in the graph and the outgoing edges from a and b
twoNodeGraph contains: a b
the children of a in twoNodeGraph are: b(AB)
the children of b in twoNodeGraph are: a(BA)